examples/CMakeLists.txt

```diff
@@ -25,3 +25,6 @@ configure_file(pipelines/images_to_kwa.pipe.in ${EXAMPLE_DIR}/pipelines/images_t
 configure_file(pipelines/images_to_kwa_split.pipe.in ${EXAMPLE_DIR}/pipelines/images_to_kwa_split.pipe @ONLY )
 configure_file(pipelines/stabilize_images.pipe.in ${EXAMPLE_DIR}/pipelines/stabilize_images.pipe @ONLY )
 configure_file(pipelines/track_features.pipe.in ${EXAMPLE_DIR}/pipelines/track_features.pipe @ONLY )
+
+install( DIRECTORY applications
+	     DESTINATION ${EXAMPLE_DIR} )
```

Comment:
I think the issue you are seeing is that `install` moves files from the build tree to the install location.  These are source files, so they first need to be copied into the build tree and then installed.  I'm not sure what the most "correct" way of doing this is, but one option is what is done for config files, see https://github.com/Kitware/kwiver/blob/master/config/CMakeLists.txt